src/PacketPay.js

```diff
@@ -98,20 +99,39 @@ module.exports = ({
         transaction: BSVPayment.transaction
       })
       if (paymentResult.status !== 'success') {
-        throw new Error('Payment not processed')
```

Comment:
Just call ` throw new Error('Payment not processed')` which can then be caught in the catch block below and the existing JSON response will be returned.